cmd/serverless-init/main.go

```diff
@@ -8,108 +8,122 @@
 package main
 
 import (
+	"context"
+	serverlessInitLog "github.com/DataDog/datadog-agent/cmd/serverless-init/log"
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/mode"
+	"github.com/DataDog/datadog-agent/comp/core"
+	"github.com/DataDog/datadog-agent/comp/core/autodiscovery"
+	"github.com/DataDog/datadog-agent/comp/core/autodiscovery/autodiscoveryimpl"
+	coreconfig "github.com/DataDog/datadog-agent/comp/core/config"
+	healthprobeDef "github.com/DataDog/datadog-agent/comp/core/healthprobe/def"
+	healthprobeFx "github.com/DataDog/datadog-agent/comp/core/healthprobe/fx"
+	"github.com/DataDog/datadog-agent/comp/core/log/logimpl"
+	"github.com/DataDog/datadog-agent/comp/core/secrets"
+	"github.com/DataDog/datadog-agent/comp/core/tagger"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/taggerimpl"
+	"github.com/DataDog/datadog-agent/comp/core/workloadmeta"
```

Comment:
Do we need this component in the serverless binary?

Reply:
replying in the thread above for autodiscovery+workloadmeta